index.html

```diff
@@ -112,6 +112,38 @@ <h2>Professional Software Development in 2018</h2>
           hodor - hodor hodor hodor. Hodor. Hodor hodor... Hodor hodor hodor hodor hodor? </p>
         <span class='expandButton'></span>
       </div>
+      <div class="article">
+        <h2>Lorem Ipsum</h2>
+        <p class="date">April 20th, 2018</p>
+        <p>I had a client review a design, and all the textarea’s were filled with ‘Lorem Ipsum’. Reply: It looks good, but we don’t understand it.</p>
+        
+        <p>Me: What don’t you understand?</p>
+        
+        <p>Reply:  The text. What language is that anyway?</p>
+
+        <p>Me: It doesn’t mean anything, it’s to fill up the textareas. Just focus on the design, not the content.</p>
+
+        <p>Reply, after a day or two: Could you please type the text from our brochure in there, we can’t review the site in a foreign language.</p>
+
+        <p><em>From</em> <a href="https://clientsfromhell.net/post/258445767/lorem-ipsum">Clients from Hell</a></p>
+        <span class='expandButton'></span>
+      </div>
+      <div class="article">
+        <h2>Why does less-watch-compiler break my heart? :(</h2>
+        <p class="date">April 25th, 2018</p>
+        <p>Dear, Lambda,</p>
+        
+        <p>Today, I was real pumped for today's project, but then all of a sudden, less-watch-compiler decided to be a total #@($@! to me. It led me to a dark alleyway of despair and robbed me of my sanity for 40 minutes. Why does it do that to me? :(</p>
+        
+        <hr> 
+
+        <p>Dear, Ronald:</p>
+
+        <p>Lorem ipsum dolor amet austin messenger bag thundercats poutine. Wolf 3 wolf moon vaporware cardigan kale chips chillwave lumbersexual chicharrones marfa typewriter palo santo craft beer.</p>
+
+        <p>Hope that helped!</p>
+        <span class='expandButton'></span>
+      </div>
     </div>
```

Comment:
Let's not insert anything here in the HTML file. We have several tools to achieve this. Higher-order functions, objects, arrays and so on.